packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -373,15 +374,15 @@ const RealAccountSignup = ({
             sessionStorage.removeItem('post_real_account_signup');
             localStorage.removeItem('real_account_signup_wizard');
         }
-        closeRealAccountSignup();
 
-        if (isNavigationFromExternalPlatform(routing_history, routes.smarttrader)) {
-            window.location = routes.smarttrader;
+        if (modal_content[getActiveModalIndex()].action === 'signup' && e) {
+            if (!is_closing_create_real_account_modal) {
+                setIsClosingCreateRealAccountModal(true);
+            }
```

Comment:
can we use a single condition to cater this
if (modal_content[getActiveModalIndex()].action === 'signup' && e && is_closing_create_real_account_modal ){
setIsClosingCreateRealAccountModal(true);
}

Reply:
Sure will edit it 👌